cpmdetail.py

```diff
@@ -16,6 +18,27 @@ def get_matches(puuid):
     return service.riotapi.get_matches(puuid, api_key, 5)
 
 
+def get_match_detail(matches, puuid):
+    matches_dict = {}
+    for match in matches:
+        match_id = match
+        data = service.riotapi.get_match_detail(match_id, puuid, api_key)
+        data['puuid'] = puuid
+        participants = data.get("info").get("participants")
+        participant = [item for (index, item) in enumerate(participants) if item.get("puuid") == puuid]
```

Comment:
If it weren't for my context on the app, I would have no idea what this is doing. Which isn't a bad thing but makes me think you want a comment above it describing what this does

Reply:
@jmirving 

Line 25: Data is storing the response of all the match data. However it is not recursive and some lists are in the dictionary.

Line 26: I add the puuid to the Data dictionary to easily reference for additional logic in this function.

Line 27: Create a Dict to filter only the participant data for the puuid from Line26.

Line 28: Is a solution I found on the Googles to index a Dict and match on value.  So the participant variable now contains only data for the matched puuid instead of data for all 10 participants.

Reply:
OK so, based on https://realpython.com/python-enumerate/ let's rename this from "index" to "count". I feel like "item" should be more descriptive but I am not sure what to call it "participant_item"?